Vital/Controllers/CycleController.cs

```diff
@@ -98,8 +98,12 @@ await _cycleService.Update(currentCycle.Id, new UpdateCycleDto()
     [HttpPut("{id:guid}")]
     [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(CycleDto))]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<IActionResult> Update(Guid id, [FromBody] UpdateCycleDto dto)
+    public async Task<IActionResult> Update([FromRoute] Guid id, [FromBody] UpdateCycleDto dto)
```

Comment:
The FromRoute does not need to be there, it is not affecting the functionality of the system